solutions/swb-reference/integration-tests/config/testEnv.yaml

```diff
@@ -21,3 +21,5 @@ defaultHostingAccountId: 'acc-827e79f0-f007-4b6a-9640-937abbb98e0d'
 ## Hosting account used for Account Creation tests
 hostAwsAccountIdParamStorePath: '/swb/testEnv/accountsTest/awsAccountId'
```

Comment:
Should these values be in [example.yaml](https://github.com/aws-solutions/solution-spark-on-aws/blob/atmikev/integration-test-hostingAcountHandlerRoleArn-update/solutions/swb-reference/integration-tests/config/example.yaml) as well? This is so other developers can set up the account integ test. 

I also noticed that this file doesn't have the values that are in `example.yaml`
```
#awsAccountId: ''
#envMgmtRoleArn: ''
#hostingAccountHandlerRoleArn: ''
#encryptionKeyArn: ''
```

Are those values no longer needed for integ tests? If not let's update `example.yaml` to remove those values.

Reply:
updated